pkg/gofr/gofr.go

```diff
@@ -321,39 +325,50 @@ func (a *App) initTracer() {
 	}
 }
 
-func (a *App) getExporter(name, host, port string) (sdktrace.SpanExporter, error) {
+func (a *App) getExporter(name, host, port, url string) (sdktrace.SpanExporter, error) {
 	var (
 		exporter sdktrace.SpanExporter
 		err      error
 	)
 
-	authHeader := strings.TrimSpace(a.Config.Get("TRACER_AUTH_KEY"))
+	authHeader := a.Config.Get("TRACER_AUTH_KEY")
 
 	switch strings.ToLower(name) {
 	case "jaeger":
-		a.container.Log("Exporting traces to jaeger.")
+		if url == "" {
+			url = fmt.Sprintf("%s:%s", host, port)
+		}
 
-		opts := []otlptracegrpc.Option{otlptracegrpc.WithInsecure(),
-			otlptracegrpc.WithEndpoint(fmt.Sprintf("%s:%s", host, port))}
+		a.container.Logf("Exporting traces to jaeger at %s", url)
+
+		opts := []otlptracegrpc.Option{otlptracegrpc.WithInsecure(), otlptracegrpc.WithEndpoint(url)}
 
 		if authHeader != "" {
 			opts = append(opts, otlptracegrpc.WithHeaders(map[string]string{"Authorization": authHeader}))
 		}
 
 		exporter, err = otlptracegrpc.New(context.Background(), opts...)
 	case "zipkin":
-		a.container.Log("Exporting traces to zipkin.")
+		if url == "" {
+			url = fmt.Sprintf("http://%s:%s/api/v2/spans", host, port)
+		}
+
+		a.container.Logf("Exporting traces to zipkin at %s", url)
 
 		var opts []zipkin.Option
 		if authHeader != "" {
 			opts = append(opts, zipkin.WithHeaders(map[string]string{"Authorization": authHeader}))
 		}
 
-		exporter, err = zipkin.New(fmt.Sprintf("http://%s:%s/api/v2/spans", host, port), opts...)
-	case traceExporterGoFr:
-		exporter = NewExporter("https://tracer-api.gofr.dev/api/spans", logging.NewLogger(logging.INFO))
+		exporter, err = zipkin.New(url, opts...)
+	case gofrTraceExporter:
+		if url == "" {
+			url = "https://tracer-api.gofr.dev/api/spans"
```

Comment:
```suggestion
			url = fmt.Sprintf("%s/api/spans", gofrTracerURL)
```

Reply:
The tracer UI and tracer API are different URLs so this cannot be done

Reply:
OK, I misunderstood then. Thanks